docs/source/docker/getting_started/first_steps.rst

```diff
@@ -362,6 +362,43 @@ you might want to change:
     )
 
 
+**Checkpoints** from your training process will be stored in the output directory.
+You can use such a checkpoint in future worker runs by copying the checkpoint to
+a `shared directory` and then passing the checkpoint filename to the container.
```

Comment:
Also here I'd give an example.

For example, you can create a new folder `/home/mymachine/lightly-shared` and copy the checkpoint (`my_checkpoint_file.ckpt`) into this folder. Then make sure to set `{SHARED_DIR}` to `/home/mymachine/lightly-shared`.